src/table/Exposure.cc

```diff
@@ -25,6 +29,14 @@ namespace lsst { namespace afw { namespace table {
 
 namespace {
 
+int const TableVersion = 2;  // current version of ExposureTable
+std::string const TableVersionKey = "TBL_VERS";  // FITS metadata key for ExposureTable version
```

Comment:
I'm going with @PaulPrice here because a short name that is unique will be illegible. There are standard ways to write long FITS header keys and I think we are simply going to have to figure out how to manage it. `VisitInfo` also saves information using some long key names.